APIWorking.js

```diff
@@ -1,65 +1,80 @@
+/**
+ * Populate year option dropdown
+ */
+const startYear = 1940;
+const thisYear = new Date().getFullYear();
+const yearDropdown = document.getElementById("yearDropdown");
+
+for (let year = startYear; year <= thisYear; year++) {
```

Comment:
Generating the dropdown this way helps avoid typos and additionally means the the most recent year will always be the current year so that it wouldn't need to updated next year and the year after that, etc.